imports/pages/devotionals/devotions.Single.js

```diff
@@ -102,6 +102,15 @@ export default class SeriesSingle extends Component {
     this.props.dispatch(headerActions.hide());
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    if (this.state.selectedIndex !== nextState.selectedIndex) return true;
+    if (nextProps.devotion.content && this.props.devotion.content) {
+      if (nextProps.devotion.content.id === this.props.devotion.content.id) return false;
```

Comment:
Added a more specific check, this way if you are navigating to a new article it will reload.